### DISCUSSION TITLE
Proposal to approach UPheno this Thursday (26th January 2023) re: Phenotype branch

### DISCUSSION THREAD

**Discussion by DitchingIt** [obo-behavior/behavior-ontology#133] [Team Posts]
# Proposal
Get agreement this Thursday (26th January 2023) from UPheno for NBO to transfer its Phenotype branch to another ontology (probably OBA).

# Objections
Unless these are raised in time, I will assume I can go ahead.

# History

1. I have noted many complications and missed expectations arising from the amalgamation of the Phenotype branch with the Process branch of NBO over years of comments.
2. There has been a live and publicly accessible discussion around the subject since 8th December 2022 at Issue #120 so this proposal is not out of the blue.
3. The use of UPheno to authorise changes to NBO in the absence of formal leadership has been openly discussed in Issue #124.
4. The [one recent plea ](https://obo-communitygroup.slack.com/archives/C01BDKWDS91/p1673605753755689)I have seen for an active Phenotype ontology would not be abandoned by this transfer, but made more fulfillable by a team more dedicated to it.

**Comment by pmidford** [comment]
I am fine with splitting the phenotypes into a separate ontology and 
passing this off to UPheno.  I assume terms will retain their NBO 
identifiers in the new ontology, or will this be handled some other way?

**Comment by DitchingIt** [comment]
@matentzn understands the process, I don't.

**Comment by matentzn** [comment]
We should deprecate the NBO ids according to standard OBO practice and add "replaced with" annotations in accordance with current principles. This is important as the identifier space is actually useful for contextualization (what ontology is this term defined in?, What is it about?) and governance (where do I go to change the label of this term).

**Comment by DitchingIt** [comment]
I will go ahead with this now - we can withdraw the request beforehand if we need to. @aclark-binghamton-edu I hope you're monitoring this and okay with it :-)

**Comment by aclark-binghamton-edu** [comment]
Yes, I am doing so and yes, ok with the plan!

On Tue, Jan 24, 2023 at 12:23 PM Ditch Townsend ***@***.***>
wrote:

> I will go ahead with this now - we can withdraw the request beforehand if
> we need to. @aclark-binghamton-edu
> <https://github.com/aclark-binghamton-edu> I hope you're monitoring this
> and okay with it :-)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/obo-behavior/teams/behavior-ontologies/discussions/1/comments/4>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADUXTFW6PQ5LUCSQLA7E2QTWUAFYDANCNFSM6AAAAAAUEISH44>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Anne B. Clark, Ph.D.
Biological Sciences
Binghamton University
Binghamton, NY 13902-6000

(607) 222-0905  (cell)
(607) 777-6521 (fax-Dept office)
(607) 777-2438 (Biol Sci office)

**Comment by DitchingIt** [comment]
@aclark-binghamton-edu @pmidford Just to let you know this is going ahead in the next few weeks hopefully. I will check which parts need to stay with us. Please post here if you spot something to retain or discuss. I will too.

**Comment by DitchingIt** [comment]
@matentzn @pmidford @aclark-binghamton-edu 

I have looked through the behavioral phenotype NBO:0000243 branch and found two classes I think are anomalies. They are related to items in the process branch and not really phenotypes in my opinion:

1. fatigue NBO:0000569 which seems to be a normal physiological thing, not abnormal, and is related to exhaustion NBO:0000472
2. asleep NBO:0000067 which I think is identical to sleeping behavior NBO:0000024

I can't find anything else to flag up. What do you think?

**Comment by matentzn** [comment]
> asleep NBO:0000067 which I think is identical to sleeping behavior NBO:0000024

Hm, this is an odd one. I think "asleep" (or "sleeping") is "phenotype" in the broad sense of the word (it will be used to describe the state of a specimen after an intervention, such as "singing a lullaby". But you are probably right - for all practical purpose, we should be able to deprecate this term and add a 'consider' relation to the sleeping behaviour branch.

Fatigue and exhaustion should be in uPheno IMO, but these may need some input from @sbello and @rays22.

**Reply by dosumis** [reply]
This is a _much_ broader use of the term phenotype than I'm comfortable with.   Sleeping is a process that most animals go through - one that is required for normal physiological functioning and that is ancient and conserved. Being asleep is just the state that an animal is in when it is asleep.

**Comment by DitchingIt** [comment]
Exhaustion is also a phase of stress adaptation so I'd rather not lose both: I don't mind accepting that fatigue as in Chronic Fatigue is a syndrome/trait but keeping exhaustion, although perhaps linking via has_related_synonym.

**Comment by matentzn** [comment]
"phase of stress adaptation" - this sounds like a complex modelling issue. It may be a process related to behavior, but not really behaviour itself. We have to make sure that "exhaustion" the NBO class really "a phase of stress adaptation" and not (what seems more likely), a character state of an organism.

**Comment by DitchingIt** [comment]
Okay - but let's leave behavioral exhaustion for another time, as it is already in the process branch and perhaps shouldn't be (but the model I have proposed makes space in the morphophysiology sub-branch, and there are indicative behaviors I think for the phases of stress adaptation). Better let fatigue go out with the phenotype branch and focus on that for now I think.

**Comment by aclark-binghamton-edu** [comment]
Some of these will need clarification, wherever they are, in terms of a definition that states how the state/phase or whatever is measured, because there is a danger of assuming species-specific aspects without stating them or stating species-specific aspects without saying they are species-specific. Exhaustion sounds to me like a physiologically measurable state.  Anyway, I agree--not a behavior even if behaviorally recognizable (probably in taxon-specific way.)

**Comment by aclark-binghamton-edu** [comment]
Re asleep or sleeping:  I could argue that "sleeping" may have typical and observable characteristics (e.g., head under wing; curled with nose onto abdomen and tail wrapped around head while breathing slowly and regularly);  being "asleep" is the corresponding physiological state.  Not voting for how to resolve, but this is in part why we included a positional/postural/body part arm of the ABO, to allow researchers to define a situation or event for their purposes in terms of what was observably happening.  Or to use those terms to investigate correspondence of observable events with an unobservable (if physiologically measurable) state.

**Comment by DitchingIt** [comment]
TLDR: Issue #126 would be a great place to explore the question of exhaustion and what else to include in the NBO process branch (Mallpress is my source for suggesting some physiology in the model). And let's go ahead and move fatigue out of NBO as a phenotype.

---
> Exhaustion sounds to me like a physiologically measurable state. Anyway, I agree--not a behavior even if behaviorally recognizable (probably in taxon-specific way.)

I agree with you in principle. But in practice, I see two issues:

1. Fatigue NBO:0000569 is the phenotype in question. The issue that has been previously agreed is to move the phenotype branch out of NBO. I think there is no disagreement on this now. My mistake was to make an argument that linked fatigue to behavioral exhaustion NBO:0000472 which is in the process branch. What appears to be happening now is a discussion about moving something out of the process branch. We need that discussion but I'm hoping we can get there via agreement on an appropriate general model #126 and set of editing conventions #122 and not tangle it up with the phenotype deprecation process which is currently under way.
2. If we take a somewhat hierarchical approach to structuring the process branch, I think we will inevitably have at least one superclass (probably quite a few) which provides a location for a cluster of clear behaviour classes, but which is not itself easily observable as a distinct behaviour. We could of course simply use a clustering label and the definition would essentially just list the subclasses. But my preference would be to try to follow an explanatory path through the hierarchy, and to use labels - even physiological ones - that get us to the clear behaviour classes lower down in a theoretically understandable (even if disputed) way. (If necessary, we can note that some particular superclass should not be used for direct annotation - as GO does.) Exhaustion may or may not be an example of this, but in a later stage I may be making that argument, so I really hope we don't throw the baby (fully and logically reworking the process branch) out with the bathwater (the phenotype branch deprecation process).
